irrep/spacegroup.py

```diff
@@ -277,8 +302,19 @@ def show(self, refUC=np.eye(3), shiftUC=np.zeros(3)):
         else: 
             json_data ["rotation_matrix_refUC"]=self.rotation
 
+        if show_k_trans:
+            kstring = "gk = ["+", ".join([parse_row_transform(r) for r in 
+                np.transpose(np.linalg.inv(self.rotation))])+"]"
+
+            if write_ref:
+                kstring += "  |   refUC:  gk = ["+", ".join([parse_row_transform(r) for r in 
+                    np.transpose(np.linalg.inv(R))])+"]"
+                
```

Comment:
In both prints:
- I'd avoid breaking the line at the middle of the `for` statement. A good option is to define an auxiliary variable `coord = ", ".join([parse_row_transform(r) for r in  np.transpose(np.linalg.inv(self.rotation))])` (obviously, doing it in a single line is  more efficient, but avoiding long and complicate one-liners may be worth it if it improves readability. Anyway, I am fine with one-liners (it's part of Python's spirit), but try to break the line out of the `for`  :).
- There should be a space surrounding the `+` sign: `"gk = ["+",` -> `"gk = [" + ",`